### PR TITLE
Update README with Summary

### PR DESCRIPTION
**Pull request by jiaweiteo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,176 +3,28 @@ layout: page
...
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+We would like to acknowledge the original source of the code, AddressBook-Level3 project created by SE-EDU initiative at https://se-education.org
```
Comment (lyeyixian) [inline README.md:30]: Is it better to change this sentence to this? 
"This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org)."
As I feel like this flows better for me.

**Review by lyeyixian — COMMENTED** [review]
I have leave some comments (actually just one haha), please check.

File: README.md [hunk 2]
```diff
@@ -3,176 +3,28 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (lyeyixian) [inline README.md:6]: Should this be linked to our repo's actions instead of the se-edu's one?

Comment (jiaweiteo) [reply README.md:6]: Ya but i dont know what badges we should add, there's quite a few different one

Comment (lyeyixian) [reply README.md:6]: no i mean the link not the badges. Now you are linking it to se-edu/addressbook-level3

Comment (lyeyixian) [reply README.md:6]: https://github.com/AY2021S1-CS2103-T14-3/tp/actions
this

File: README.md (hunk 1 above)
Comment (jiaweiteo) [reply README.md]: Changed

File: README.md (hunk 2 above)
Comment (jiaweiteo) [reply README.md]: Changed

**Review by lyeyixian — APPROVED** [review]

**Pull request merged**